src/monitoring/MonitoringService.ts

```diff
@@ -96,9 +102,15 @@ export default class MonitoringService {
       }
     )
 
+    this.worker.on("error", (error: Error) => {
+      logger.error({
+        message: "Monitoring service has errored",
+        error,
+      })
+    })
```

Comment:
no issues on using `logger.error` here, just a reminder that this should ideally ping the alarms channel or call on-call as this implies the service has already gone down